src/Components/Autocomplete.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {
+  useState, useMemo, useCallback, useRef,
+} from 'react';
+import { debounce } from 'lodash';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  onSelected: (person: Person) => void;
+};
+
+export const Autocomplete: React.FC<Props> = React.memo(({
+  people, onSelected,
+}) => {
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [focus, setFocus] = useState(false);
+
+  const searchInput = useRef<HTMLInputElement>(null);
+
+  const applyQuery = useCallback(
+    debounce(setAppliedQuery, 1000),
+    [],
+  );
+
+  const filteredPeople = useMemo(() => {
+    return people.filter(person => person.name
+      .toLocaleLowerCase().includes(appliedQuery.toLowerCase()));
+  }, [appliedQuery]);
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+    applyQuery(event.target.value);
+  };
+
+  const handleSelectedPerson = (
+    event: React.MouseEvent<HTMLAnchorElement>,
+    person: Person,
+  ) => {
+    event.preventDefault();
+    setQuery(person.name);
+    setFocus(false);
+
+    if (searchInput.current) {
+      searchInput.current.blur();
+    }
+
+    onSelected(person);
+  };
+
+  return (
+    <div className="dropdown is-active">
```

Comment:
The dropdown is always active because the dropdown's 'is-active' class is hard-coded. Use the focus state to conditionally add the 'is-active' class